dataclients/kubernetes/ingressdefinitions_test.go

```diff
@@ -38,16 +65,76 @@ func TestGetTargetPort(t *testing.T) {
 							Port: 80,
 						},
 					},
-				}},
+				},
+			},
 			svcPort:     definitions.BackendPort{Value: 80},
 			expected:    nil,
 			errExpected: true,
 		},
+		{
+			name: "multiple service ports, by value",
```

Comment:
please use the new fixture test suite

Reply:
I added tests covering these cases using the new fixture test suite in f5fda805e43059c699043f9a1d81e6b0f6ccc7cc

This test was testing `getServicePort()` specifically as it replaced `getTargetPort()`. Would you like me to remove these tests and only rely on the new tests instead?

Reply:
yes only the fixture tests

Reply:
Done. I put the tests into `dataclients/kubernetes/testdata/ingress/service-ports`. Is there any naming convention to follow? Happy to move the tests into `named-ports` or `ingress-data` if you see them fit one of these.